components/dashboard/dashboard.go

```diff
@@ -1,244 +1,245 @@
-// Package dashboard provides utility functions to config Open Data Hub Dashboard: A web dashboard that displays
-// installed Open Data Hub components with easy access to component UIs and documentation
-// +groupName=datasciencecluster.opendatahub.io
+// // Package dashboard provides utility functions to config Open Data Hub Dashboard: A web dashboard that displays
+// // installed Open Data Hub components with easy access to component UIs and documentation
+// // +groupName=datasciencecluster.opendatahub.io
 package dashboard
 
-import (
-	"context"
-	"errors"
-	"fmt"
-	"path/filepath"
-
-	"github.com/go-logr/logr"
-	operatorv1 "github.com/openshift/api/operator/v1"
-	corev1 "k8s.io/api/core/v1"
-	k8serr "k8s.io/apimachinery/pkg/api/errors"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"sigs.k8s.io/controller-runtime/pkg/client"
-	logf "sigs.k8s.io/controller-runtime/pkg/log"
-
-	dsciv1 "github.com/opendatahub-io/opendatahub-operator/v2/apis/dscinitialization/v1"
-	"github.com/opendatahub-io/opendatahub-operator/v2/components"
-	"github.com/opendatahub-io/opendatahub-operator/v2/pkg/cluster"
-	"github.com/opendatahub-io/opendatahub-operator/v2/pkg/deploy"
-)
-
-var (
-	ComponentNameUpstream = "dashboard"
-	PathUpstream          = deploy.DefaultManifestPath + "/" + ComponentNameUpstream + "/odh"
-
-	ComponentNameDownstream = "rhods-dashboard"
-	PathDownstream          = deploy.DefaultManifestPath + "/" + ComponentNameUpstream + "/rhoai"
-	PathSelfDownstream      = PathDownstream + "/onprem"
-	PathManagedDownstream   = PathDownstream + "/addon"
-	OverridePath            = ""
-	DefaultPath             = ""
-)
-
-// Verifies that Dashboard implements ComponentInterface.
-var _ components.ComponentInterface = (*Dashboard)(nil)
-
-// Dashboard struct holds the configuration for the Dashboard component.
-// +kubebuilder:object:generate=true
-type Dashboard struct {
-	components.Component `json:""`
-}
-
-func (d *Dashboard) Init(ctx context.Context, platform cluster.Platform) error {
```

Comment:
This is should not be reverted, it is sort of regression.

Reply:
Updated